Description: Upstream changes introduced in version 1.0.0.99-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 jrl-walkgen (1.0.0.99-1) lucid; urgency=low
 .
   * Initial work.
 .
 The person named in the Author field signed this changelog entry.
Author: Thomas Moulard <thomas.moulard@gmail.com>

---
Disable unit testing as unit test compilation requires non-free packages.
Last-Update: 2010-09-29

--- jrl-walkgen-1.0.0.99.orig/CMakeLists.txt
+++ jrl-walkgen-1.0.0.99/CMakeLists.txt
@@ -124,7 +124,7 @@ ENDIF(CMAKE_BUILD_TYPE MATCHES "RELEASE"
 # To deal with the headers.
 #--------------------------
 add_subdirectory(src)
-add_subdirectory(unitTesting)
+#add_subdirectory(unitTesting)

 # Prepare the documentation
 # -------------------------
